Jobsway2goMvc/Controllers/CollectionsController.cs

```diff
@@ -207,7 +207,7 @@ public async Task<IActionResult> DeleteConfirmed(int id)
             
             if (collection != null)
             {
-                if (collection.Job.Count() > 0)
+                if (collection.Jobs.Count() > 0)
                 {
                     ViewBag.DeleteWarning = "The collection you’re about to delete is not empty.";
```

Comment:
Check if jobs are included when creating collection. If not, use `.Include()`

Reply:
Added `.Include()` as suggested.